extensions/wikia/Rail/styles/RailElement.scss

```diff
@@ -0,0 +1,269 @@
+@import "skins/shared/color";
+
+$black: #000;
+$white: #FFF;
+$color-divider: #E6EBF2;
+$color-rail-background: mix($color-page, $black, 90%);
+@if $is-dark-wiki {
+    $color-rail-background: mix($color-page, $white, 90%);
+}
+
+
+@mixin wikia-rail-element() {
+    background-color: $color-rail-background;
+    padding: 20px;
+
+    zoom: 1;
```

Comment:
Maybe scss-lint doesn't say that about mixins, but attributes should be listed in an alphabetical order.